src/ansys/dpf/post/simulation.py

```diff
@@ -658,11 +667,9 @@ def _build_selection(
                 selection.select_elements(elements=element_ids)
         elif node_ids is not None:
             if location != locations.nodal:
```

Comment:
Needed if requested if the input is a nodal selection, but requires_manual_averaging returns true.